scripts/init/database/migrations/001.do.sql

```diff
@@ -13,7 +13,7 @@ CREATE TABLE organizations (
 
 /* TODO: need policies unique constraint on org_id, name */
 CREATE TABLE policies (
-  id          SERIAL UNIQUE,
+  id          VARCHAR(128) UNIQUE,
```

Comment:
I think that as long as we do not have a tag we may not need to add migrations.

I'm not sure if investing time in targeting a specific library for migration (ie: postgrator) is a better idea then some documentation on how to upgrade in case we do a breaking change. 

(We may still want to user postgrator for our own fixtures. Having a migration may be useful to us and as an example that complements the changelog)

Given that this is kind of a big topic, I think it's better if we shift the discussion [here](https://github.com/nearform/labs-authorization/issues/268)?